.changeset/sweet-planets-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+'@apollo/server': patch
+---
+
+🐛 Bug Fix for Apollo Server Landing Pages on Safari. A Content Security Policy was added to our landing page html so that Safari can run the inline scripts we use to call the Embedded Sandbox & Explorer
```

Comment:
```suggestion
🐛 Bug Fix for Apollo Server Landing Pages on Safari. A Content Security Policy was added to our landing page html so that Safari can run the inline scripts we use to call the Embedded Sandbox & Explorer.
```